### PR TITLE
[Update] Preparations for U10 release

### PR DESCRIPTION
**Pull request by yo1995**
This PR will update related files for U10 release.

- [x] Root README
    - Update the minimum version
    - Remove `latest` from URL pattern to cope with the new doc site
    - Adjust wordings for CocoaPods
- [x] Toolkit to latest commit
- [x] Project marketing version to 100.10.0
- [x] Release note (see below)
- [ ] Tag the release
- [x] CocoaPods

### CODE REVIEW

**Comment by yo1995** [comment]
The draft release note for U10 tag
---

The following samples have been added:

• Augmented reality > View hidden infrastructure in AR
• Layers > Add ENC exchange set
• Geometry > Buffer list
• Route and directions > Navigate route with rerouting

The following samples have been updated:

• Layers > Edit KML groud overlay
• Layers > Export tiles
• Layers > ArcGIS vector tiled layer (custom style)
• Layers > Group layers
• Layers > Apply mosaic rule to rasters
• Scenes > Add integrated mesh layer
• Augmented reality > Flyover AR
• Scenes > Create terrain from local tile package
• Maps > Change basemap
• Geometry > Spatial operations

In addition, this update includes the following changes:

• Added Dark Mode adaption
• Slightly improved coding style
• Updated build phases and path to the new ArcGIS Framework
• Incorporated API key and new basemap constructors

**Review by yo1995 — COMMENTED** [review]
Please share your thoughts on the following stuff! 

Once the toolkit is released, I'll update to its latest commit, and merge this PR into `v.next`
Then I'll create the PR to merge `v.next` into `master`, and tag the release.

**Comment by yo1995** [comment]
Currently we cannot build a preview sample code documentation site based on `v.next`, as it will need to wait until the site go live. 😕

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ArcGIS Runtime SDK for iOS Samples [![](https://user-images.githubusercontent.com/2257493/54144188-6fe0fc00-43e8-11e9-8cf5-229af80f604a.png)](https://itunes.apple.com/us/app/arcgis-runtime-sdk-for-ios/id1180714771)
 ==========================
 
-This repository contains Swift sample code demonstrating the capabilities of [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/latest/) and how to use them in your own app. The project that can be opened in Xcode and run on a simulator or a device. Or you can [download the app from the App Store](https://itunes.apple.com/us/app/arcgis-runtime-sdk-for-ios/id1180714771) on your iOS device.
+This repository contains Swift sample code demonstrating the capabilities of [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/) and how to use them in your own app. The project that can be opened in Xcode and run on a simulator or a device. Or you can [download the app from the App Store](https://itunes.apple.com/us/app/arcgis-runtime-sdk-for-ios/id1180714771) on your iOS device.
```
[the remark this answers is not in the document]
Comment (mikewilburn) [reply README.md:4]: That's correct. The new links should redirect, but thanks for taking care of this one!

Comment (dg0yal) [inline README.md:4]:
```suggestion
This repository contains Swift sample code demonstrating the capabilities of [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/) and how to use them in your own app. The project can be opened in Xcode and run on a simulator or a device. Or you can [download the app from the App Store](https://itunes.apple.com/us/app/arcgis-runtime-sdk-for-ios/id1180714771) on your iOS device.
```

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ The *ArcGIS Runtime SDK Samples app* has a *Target SDK* version of *12.0*, meani
 1. **Fork** and then **clone** the repository
 1. **Install** the ArcGIS Runtime SDK for iOS by running the `pod install` command in the folder where you cloned this repository
 1. **Open** the `arcgis-ios-sdk-samples.xcworkspace` **workspace** file
-1. **Select** the `arcgis-ios-sdk-samples` project node, go to the `Build Phases` tab, and **delete** the phases for `Embed Frameworks` and `Strip Frameworks` (these phases conflict with CocoaPods and are only required when using the installed SDK as described in the previous section)
+1. **Select** the `arcgis-ios-sdk-samples` project node, go to the `Build Phases` tab, and **delete** the phases for `Embed Frameworks`. (the phase conflicts with CocoaPods and are only required when using the installed SDK as described in the previous section)
```
Comment (dg0yal) [inline README.md:54]:
```suggestion
1. **Select** the `arcgis-ios-sdk-samples` project node, go to the `Build Phases` tab, and **delete** the phases for `Embed Frameworks`. (This phase conflicts with CocoaPods and is only required when using the installed SDK as described in the previous section)
```

**Review by dg0yal — CHANGES_REQUESTED** [review]

**Review by dg0yal — APPROVED** [review]

**Review by vquach2404 — APPROVED** [review]

**Comment by dg0yal** [comment]
Let’s go 
Let’s go

**Comment by yo1995** [comment]
I've updated the toolkit to U10!

**Review by dg0yal — APPROVED** [review]

**Review by vquach2404 — APPROVED** [review]

**Pull request merged**